AwsCryptographicMaterialProviders/runtimes/java/build.gradle.kts

```diff
@@ -224,18 +224,21 @@ tasks.test {
     // This will show System.out.println statements
     testLogging.showStandardStreams = true
 
-    testLogging {
-        lifecycle {
-            events = mutableSetOf(org.gradle.api.tasks.testing.logging.TestLogEvent.FAILED, org.gradle.api.tasks.testing.logging.TestLogEvent.PASSED, org.gradle.api.tasks.testing.logging.TestLogEvent.SKIPPED)
-            exceptionFormat = org.gradle.api.tasks.testing.logging.TestExceptionFormat.FULL
-            showExceptions = true
-            showCauses = true
-            showStackTraces = true
-            showStandardStreams = true
-        }
-        info.events = lifecycle.events
-        info.exceptionFormat = lifecycle.exceptionFormat
-    }
+    // Disable Verbose test logging to make the LocalCMCTests less flaky.
+    // It is hard to log 300k messages to a terminal,
+    // particularly if 10 threads are concurrently writing to the terminal.
+//    testLogging {
+//        lifecycle {
+//            events = mutableSetOf(org.gradle.api.tasks.testing.logging.TestLogEvent.FAILED, org.gradle.api.tasks.testing.logging.TestLogEvent.PASSED, org.gradle.api.tasks.testing.logging.TestLogEvent.SKIPPED)
+//            exceptionFormat = org.gradle.api.tasks.testing.logging.TestExceptionFormat.FULL
+//            showExceptions = true
+//            showCauses = true
+//            showStackTraces = true
+//            showStandardStreams = true
+//        }
+//        info.events = lifecycle.events
+//        info.exceptionFormat = lifecycle.exceptionFormat
+//    }
```

Comment:
Why not just remove this instead of commenting?

Reply:
Because if a test fails,
it is very helpful to know what it is.
Which this lifecycle logging helps with.

Reply:
Then maybe use a flag or disable the params that control this? Maybe this can help: https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/testing/logging/TestExceptionFormat.html#SHORT 

I'm afraid that by commenting it out, we are creating a precedence that it's okay to comment out code for "future" use case. And have seen this in our code base. If nothing else, we can remove this and add link to the gradle doc.